### PR TITLE
Header scroll lag fix

### PR DESCRIPTION
**Pull request by EmmanuelObua**
- Removed scroll.css and added its content to the responsive.css
- Removed unwanted CSS styles linked before from index.html
- Modified CSS style files to be cross-platform compatible

### CODE REVIEW

File: assets/css/responsive.css [hunk 1]
```diff
@@ -166,22 +172,10 @@
 	}
 
 	#features .yellow-phone img {
-		top: 37%;
-		left: -45%;
+		top: 40%;
+		left: -62%;
```
Comment (mentumtech) [inline assets/css/responsive.css:176]: @EmmanuelObua why the change in the phone graphic?

**Review by mentumtech — COMMENTED** [review]
One question

File: assets/css/responsive.css (hunk 1 above)
Comment (EmmanuelObua) [reply assets/css/responsive.css:176]: Adding content-visibility CSS property on the section dom element was overlapping the cards on the margins on mobile, so I had to add some paddings on the left and right of the section element and then recenter the SVG of the yellow-phone by modifying the top and left elements.
